carbonj.service/build.gradle

```diff
@@ -99,12 +99,10 @@ configurations {
 //    compile.exclude group: 'org.apache.logging.log4j', module: 'log4j-to-slf4j'
 }
 
-
-
 task rpm(type: Rpm) {
 	packageName project.name
-	version version
-	release '15'
+	version "${project.hasProperty('versionSuffix') ? version + '_' + project.versionSuffix : version}"
```

Comment:
does that add branch build capabilities?

Reply:
Yes for feature branches